docs/developing.md

```diff
@@ -4,41 +4,45 @@
 
 On all platforms, the minimum requirements are:
 
-- Git client and command line tools
+- [Docker](https://docs.docker.com/engine/install/)
+- [Docker Compose](https://docs.docker.com/compose/install/)
 - [.NET 6.0 SDK](https://dotnet.microsoft.com/download/dotnet/6.0)
```

Comment:
Is it really needed?

Reply:
yes. I was getting some errors. I can check later if we can get rid of them somehow.

Reply:
https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/issues/1723